### ISSUE TITLE
Remove implicit adding of `Input` suffix

### ISSUE THREAD

**Issue by meiblorn** [dotansimha/graphql-code-generator#3998]
https://github.com/dotansimha/graphql-code-generator/blob/f7d17324e7b24efbf46815139964ae042dd34657/packages/plugins/java/kotlin/src/visitor.ts#L125

Hi, I am developing a Java / Kotlin solution. And my problem is that I can't use `Input` endings in my schema for the `input` types because `codegen` utility translates `input` type names into `SomeNameInputInput`. 
So, generally speaking, it duplicates the `Input` ending. 

Can you make this optional? Or add some logic that checks "if type ends with `Input` word, then do not add additional `Input` word"?

Steps to reproduce:
1) Open `https://graphql-code-generator.com/`
2) Select `Java --> Types (Enum & Input)` in dropdown
3) Put this in the textarea on the left side

```graphql
input CreateUserInput {
   userName: CreateUserNameInput
}

input CreateUserNameInput {
  given: String!
  family: String!
}
```
4) Find `CreateUserNameInputInput` in the right side textarea

**Comment by meiblorn** [comment]
Implemented a fix. See MR above.

**Comment by meiblorn** [comment]
Any update on this?

**Comment by meiblorn** [comment]
@dotansimha , @ardatan ?

**Comment by dotansimha** [comment]
Thanks @meiblorn . Your PR was merged. We'll release a new stable version soon. In the meantime, you can use the latest alpha version.

**Comment by dotansimha** [comment]
Available in `v1.14.0`.
